openedx/features/caliper_tracking/tests/expected/edx.special_exam.practice.attempt.created.json

```diff
@@ -0,0 +1,53 @@
+{
+  "@context": "http://purl.imsglobal.org/ctx/caliper/v1p1",
+  "action": "NavigatedTo",
```

Comment:
Is this action appropriate?

Reply:
Yep, If we considered the event as `NavigationEvent` then that's right.

Reply:
No. It is not appropriate for this event.

Reply:
I have changed the `event_type` and `action` as tehreem suggested. So now I think it's correct.